tests/test_end_to_end_run.py

```diff
@@ -42,6 +44,19 @@ def test_end_to_end(s3):
     os.system(f"python data_linter/command_line.py --config-path {config_path}")
 
 
+def test_end_to_end_no_creds_error():
+
+    from data_linter.validation import run_validation
+    from botocore.exceptions import ClientError
+
+    test_folder = "tests/data/end_to_end1/"
+    config_path = os.path.join(test_folder, "config.yaml")
+    with open(config_path) as f:
+        config = yaml.safe_load(f)
```

Comment:
it's unclear what these two lines are needed for - as flake8 says, `config` is defined but not used